proto/vmexec.proto

```diff
@@ -2,6 +2,21 @@ syntax = "proto3";
 
 package vmexec;
 
+import "proto/remote_execution.proto";
+
+message FileSystemLayout {
+  build.bazel.remote.execution.v2.Tree inputs = 1;
+}
+
+message CASFS {
```

Comment:
nit: consider naming this `CASFSConfiguration` (or remove in favor of just `FileSystemLayout` -- see below)

Reply:
Renamed to CASFSConfiguration. I think it makes sense for it to have an explicit configuration field to make it obvious that it's related to configuring the CASFS filesystem.

Reply:
Renamed to CASFSConfiguration. I like having an explicit field to make it obvious that this relates to CASFS setup.